testing/src/main/java/org/oppia/android/testing/TextInputAction.kt

```diff
@@ -0,0 +1,40 @@
+package org.oppia.android.testing
+
+import android.content.Context
+import android.view.View
+import androidx.annotation.StringRes
+import com.google.android.material.textfield.TextInputLayout
+import org.hamcrest.Description
+import org.hamcrest.Matcher
+import org.hamcrest.TypeSafeMatcher
+import javax.inject.Inject
+
+class TextInputAction @Inject constructor(
```

Comment:
Add `kdoc` to help developers understand what does this class do and when to use it. 
Refernce - `EditTextInputAction`

Reply:
@anandwana001 
Will it be correct to write

Action for accessing HasErrorTest and HasNoErrorTest Functions.

This is needed because HasErrorTest Function and HasNoErrorTest
Function can now be accessed from this file wherever needed
Without Implementing it.

Reply:
**or reason is same as EditTextInputAction** File
This is needed because Robolectric doesn't seem to properly input digits for text views using
'android:digits' or other filters. See https://github.com/robolectric/robolectric/issues/5110
for specifics.

Reply:
@anandwana001 
PTAL

Reply:
Sounds good, need for robolectric

Reply:
@Akshat-gour

Reply:
done